### DISCUSSION TITLE
Is vscode cheating?

### DISCUSSION THREAD

**Discussion by SakuOrdrTab** [codewars/codewars.com#3114] [General]
As I try the harder katas, it is getting harder and harder to code with the online interface..

Is it considered cheating if I implement the code in vscode and then just copy-paste it to codewars?

I have not come up with more sophisticated debug options in the online interface except print statements and it is much easier to code in vscode, but I can understand that estimating whether this is own code is almost impossible, if I just paste my code from vscode to the site?

**Comment by Blind4Basics** [comment]
> Is it considered cheating if I implement the code in vscode and then just copy-paste it to codewars?

no problem with that!

**Comment by SakuOrdrTab** [comment]
That is clear, thanks!
